src/pages/developing/web-components-tutorial/step-1.mdx

```diff
@@ -0,0 +1,766 @@
+---
+title: 1. Installing Carbon
+description:
+  Welcome to Carbon! This tutorial will guide you in creating a Vanilla JS/HTML
+  app using Web Components from the Carbon Design System.
+tabs: ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5']
+---
+
+import Preview from 'components/Preview';
+
+<PageDescription>
+
+Starting from a base `create-vite` app, created using the `Vanilla` and
+`Javascript` options, let's install Carbon and begin using Carbon components. By
+the end you will have a Vanilla app that uses the UI Shell to navigate between
+pages.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Fork, clone and branch</AnchorLink>
+<AnchorLink>Build and start</AnchorLink>
+<AnchorLink>Install Carbon</AnchorLink>
+<AnchorLink>Install Sass</AnchorLink>
+<AnchorLink>A working Carbon button</AnchorLink>
+<AnchorLink>Add UI Shell</AnchorLink>
+<AnchorLink>Push to GitHub</AnchorLink>
+
+</AnchorLinks>
+
+## Preview
+
+A [preview](https://solid-carnival-1pg38np.pages.github.io/) of what you will
+build:
+
+<Preview
+  height="200"
+  title="Carbon Next Tutorial Step 1"
+  src="https://carbon-tutorial-nextjs-git-v11-next-step-2-carbon-design-system.vercel.app/"
+  frameborder="no"
+  allowtransparency="true"
+  allowfullscreen="true"
+/>
+
+## Fork, clone and branch
+
+This tutorial has an accompanying GitHub repository called
+[carbon-tutorial-web-components](https://github.com/carbon-design-system/carbon-tutorial-web-components)
```

Comment:
@lee-chase I believe this is because this repo is internal. The setting is grayed out for me too so I can't turn it on. Any reason we can't make the repo public, or did you want to wait?